### PR TITLE
Feat/kbdlight

### PR DESCRIPTION
**Pull request by sermart1234**
Get FaN Speed

### CODE REVIEW

File: huawei-wmi.c [hunk 1]
```diff
@@ -922,6 +924,79 @@ static void huawei_wmi_power_unlock_exit(struct device *dev)
...
+static int huawei_wmi_fan_speed_get(u8 num, int *rpm)
+{
+	u8 ret[0x100] = { 0 };
+	int err;
+	
+	union hwmi_arg arg;
+	arg.cmd = FAN_SPEED_GET;
+	arg.args[2] = num;
+
+	err = huawei_wmi_cmd(arg.cmd, ret, 0x100);
```
Comment (egormanga) [inline huawei-wmi.c:938]: What exactly is the `0x100` constant? If it's something meaningful, then maybe `#define` it?

File: huawei-wmi.c [hunk 2]
```diff
@@ -922,6 +924,79 @@ static void huawei_wmi_power_unlock_exit(struct device *dev)
...
+	if (err)
```
Comment (egormanga) [inline huawei-wmi.c:939]: In which cases could this happen? Do you have any examples, or rather do you need someone to test that on an unsupported laptop model?

**Review by egormanga — APPROVED** [review]
Seems fine otherwise.
Disclaimer: untested.

File: huawei-wmi.c (hunk 1 above)
Comment (aymanbagabas) [reply huawei-wmi.c:938]: `0x100` is the buffer size the HWMI interface allocates. I think it would be a good idea to `#define` it somewhere

**Review by egormanga — APPROVED** [review]

**Pull request merged**